### PR TITLE
Update go modules

### PR DESCRIPTION
**Pull request by cfergeau**

### CODE REVIEW

**Comment by praveenkumar** [comment]
@cfergeau can we remove snyk commit because if product sec require this also to be added downstream then we might need to use openshif-ci?

**Comment by cfergeau** [comment]
> @cfergeau can we remove snyk commit because if product sec require this also to be added downstream then we might need to use openshift-ci?

I'll remove the snyk commit because I haven't been able to make the snyk token work at the org level, only for my personal repos.
If snyk is important, I'd much rather to have this upstream in github actions rather than buried deep in some cryptic openshift CI jobs.

File: go.mod [hunk 1]
```diff
@@ -1,53 +1,52 @@
 module github.com/crc-org/routes-controller
 
-go 1.18
+go 1.20
```
Comment (praveenkumar) [inline go.mod:3]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**